### PR TITLE
Removing Deadline field

### PR DESCRIPTION
**Pull request by batuhanovski**
Deadline field is deleted from 
-GoalPage, Add Goal Page, Edit Goal Page
-Sub-Goal Page vice versa.

### CODE REVIEW

File: application/frontend/src/pages/LinkEntity.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export function LinkEntity() {
             key: 'entityType',
         },
         {
-            title: 'Delete',
+            title: 'Link',
```
Comment (aberatcan) [inline application/frontend/src/pages/LinkEntity.tsx:45]: what is the purpose of this change @batuhanovski ?

Comment (batuhanovski) [reply application/frontend/src/pages/LinkEntity.tsx:45]: it was just an old correction :D It is not related to "deadline feature".

Comment (aberatcan) [reply application/frontend/src/pages/LinkEntity.tsx:45]: OK 👍

**Review by aberatcan — APPROVED** [review]
